<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCEAN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="epsilonC">
     <attribute name="title">
      <string>EpsilonC</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="model_forEpsC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Модель:</string>
      </property>
      <widget class="QRadioButton" name="second_model_epsC">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="first_model_epsC">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#1</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="saltiness_forEpsC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>521</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Солёность:</string>
      </property>
      <widget class="QSlider" name="saltiness_slider_epsC">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="saltiness_textEdit_epsC">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="temperature_forEpsC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>521</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Температура:</string>
      </property>
      <widget class="QRadioButton" name="default_radiobutton_epsC">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>По умолчанию</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="choose_radiobutton_epsC">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="temperatures_textEdit_epsC">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Ввод через запятую</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="kappaFromTheta">
     <attribute name="title">
      <string>Kappa(Theta)</string>
     </attribute>
     <widget class="QGroupBox" name="model_forkappafromtheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Модель:</string>
      </property>
      <widget class="QRadioButton" name="second_model_kappatheta">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="first_model_kappatheta">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#1</string>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="saltiness_forkappafromtheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Солёность:</string>
      </property>
      <widget class="QSlider" name="saltiness_slider_kappatheta">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="saltiness_textEdit_kappatheta">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="temperature_forkappafromtheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Температура:</string>
      </property>
      <widget class="QSlider" name="temperature_slider_kappatheta">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="temperature_textEdit_kappatheta">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="alpha_forkappafromtheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Alpha:</string>
      </property>
      <widget class="QSlider" name="alpha_slider_kappatheta">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="alpha_textEdit_kappatheta">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="lamda_forkappafromtheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Lamda:</string>
      </property>
      <widget class="QRadioButton" name="default_radiobutton_kappatheta">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>По умолчанию</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="choose_radiobutton_kappatheta">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="lamda_textEdit_kappatheta">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Ввод через запятую</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="kappaFromT">
     <attribute name="title">
      <string>Kappa(T)</string>
     </attribute>
     <widget class="QGroupBox" name="model_kappafromT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Модель:</string>
      </property>
      <widget class="QRadioButton" name="second_model_kappaT">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="first_model_kappaT">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>#1</string>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="saltiness_kappafromT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Солёность:</string>
      </property>
      <widget class="QSlider" name="saltiness_slider_kappaT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="saltiness_textEdit_kappaT">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="theta_kappafromT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Theta:</string>
      </property>
      <widget class="QSlider" name="theta_slider_kappaT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="theta_textEdit_kappaT">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="alpha_kappafromT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Alpha:</string>
      </property>
      <widget class="QSlider" name="alpha_slider_kappaT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="alpha_textEdit_kappaT">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="lamda_kappafromT">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Lamda:</string>
      </property>
      <widget class="QRadioButton" name="default_radiobutton_kappaT">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>По умолчанию</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="choose_radiobutton_kappaT">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="lamda_textEdit_kappaT">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Ввод через запятую</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saving_checkBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить в файл</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
